packages/vscode-extension/package.nls.json

```diff
@@ -187,7 +187,8 @@
     "teamstoolkit.handlers.grantPermissionSucceededV3": "Added account: '%s' as the Teams app owner.",
     "teamstoolkit.handlers.grantPermissionWarning": "If added user cannot access Azure resources, you need to manually setup access policy via Azure portal. ",
     "teamstoolkit.handlers.grantPermissionWarningSpfx": "If added user is not a SharePoint App Catalog site admin, you need to manually setup access policy via SharePoint admin center. ",
-    "teamstoolkit.handlers.installAdaptiveCardExt": "To preview and debug Adaptive Cards, we recommend to use the \"Adaptive Card Studio\" extension.",
+    "teamstoolkit.handlers.installAdaptiveCardExt": "To preview and debug Adaptive Cards, we recommend to use the \"Adaptive Card Previewer\" extension.",
```

Comment:
Please extract 'Adaptive Card Previewer' string or add comment to let translators know if no need to localize.

Reply:
Thanks for the info, are there such example for where to add comment or which file to extract string to?

Reply:
You can take `_teamstoolkit.commandsTreeViewProvider.previewTitle.comment` as an example.